src/main/java/it/pagopa/pagopa/apiconfig/controller/CreditorInstitutionsController.java

```diff
@@ -78,6 +85,45 @@
         return ResponseEntity.ok(creditorInstitutionsService.getCreditorInstitution(creditorInstitutionCode));
     }
 
+    @Operation(summary = "Create creditor institution", security = {@SecurityRequirement(name = "ApiKey")}, tags = {"Creditor Institutions"})
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "201", description = "OK.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = CreditorInstitutionDetails.class))),
+            @ApiResponse(responseCode = "403", description = "Forbidden client error status.", content = @Content(mediaType = "application/json", schema = @Schema())),
+            @ApiResponse(responseCode = "404", description = "Not Found", content = @Content(mediaType = "application/json", schema = @Schema())),
+            @ApiResponse(responseCode = "409", description = "Conflict", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ProblemJson.class))),
+            @ApiResponse(responseCode = "429", description = "Too many requests", content = @Content(mediaType = "application/json", schema = @Schema())),
+            @ApiResponse(responseCode = "500", description = "Service unavailable.", content = @Content(mediaType = "application/json", schema = @Schema(implementation = ProblemJson.class)))})
+    @PostMapping(value = "", produces = {"application/json"})
+    public ResponseEntity<CreditorInstitutionDetails> createCreditorInstitution(@RequestBody @Valid @NotNull CreditorInstitutionDetails creditorInstitutionDetails) {
+        return ResponseEntity.status(HttpStatus.CREATED).body(creditorInstitutionsService.createCreditorInstitution(creditorInstitutionDetails));
```

Comment:
Dis-aligned `409` ?  vs https://pagopa.atlassian.net/wiki/spaces/PAG/pages/374375153/Design+Review+API+configurazione+pagoPA _Errori
400 Bad request
404 creditorinstitutionscode not found
403 Forbidden
429 Too many requests
500 Internal Server Error_

Reply:
I updated the _design review_ page